eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -37,9 +37,8 @@ function Set-GitHubAPIParameters ($members,  $parameterName, $parameters, $allow
 
 function Get-GitHubPullRequests {
   param (
-    [Parameter(Mandatory = $true)]
+    $RepoNameWithOwner,
```

Comment:
```suggestion
    $RepoOwner,
    $RepoName,
    $RepoId = "$RepoOwner/$RepoName",
```
We should change it to like this and note that the order is important as RepoId needs to be after the owner/name in order to use those values in the default.